### ISSUE TITLE
[BUG] Bigger font sizes make menu text go off screen

### ISSUE THREAD

**Issue by NightMachinery** [ahrm/sioyek#565]
Using
```
font_size 26
```

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/36224762/209397427-3333d65a-3ba1-40fc-8477-aee193224848.png">

As you can see, the hotkeys are going off the displayed window even though there is plenty of blank space before them.

**Comment by ahrm** [comment]
Can't reproduce it on windows or linux. Seems to be a mac-specific problem, unfortunately it is hard for me to debug because I don't have a mac machine.

**Comment by NightMachinery** [comment]
> Can't reproduce it on windows or linux. Seems to be a mac-specific problem, unfortunately it is hard for me to debug because I don't have a mac machine.

Thanks for investigating.

**Comment by dgfl-gh** [comment]
Happens for me too unfortunately, indeed on macOS. It doesn't seem to depend on font size though, it happens with any font size.
I've noticed the issue seems to be precisely that the scroll bar is rendered on top of the keybindings, so with larger font sizes it's actually less noticeable. See screenshots.

font size 5:
<img width="830" alt="CleanShot 2023-01-15 at 11 39 47@2x" src="https://user-images.githubusercontent.com/43384369/212536018-cd855119-c03e-474a-bc65-013bc78fe4f0.png">

font size 12:
<img width="807" alt="CleanShot 2023-01-15 at 11 40 47@2x" src="https://user-images.githubusercontent.com/43384369/212536052-51c126b2-2816-48d9-b9db-897756e01f1d.png">

font size 26:
<img width="808" alt="CleanShot 2023-01-15 at 11 41 20@2x" src="https://user-images.githubusercontent.com/43384369/212536076-6d71cf9e-89c2-4fa2-9637-2fc8b2fe253e.png">

**Comment by dgfl-gh** [comment]
If I can do anything to help debug this, let me know. I understand your views on toolbars (i.e.  https://github.com/ahrm/sioyek/issues/588), and while I definitely don't share them, for me this command window would be good-enough discoverability (it would be amazing if it had a description of the commands showing in an overview/popup and if that description could be matched by the command search bar).
Unfortunately as it is it's not as useful.

**Comment by ahrm** [comment]
Is this fixed in the development branch?

**Comment by NightMachinery** [comment]
@ahrm Yes, AFAIR I completely removed the workaround code when rebasing on dev, and things worked great without it.
